config.toml

```diff
@@ -225,16 +225,21 @@ pre = "<i class='fas fa-blog'></i>"
 weight = 11
 [[menu.main]]
 name = "Environments"
+url = "/environments/"
 pre = "<i class='fas fa-store'></i>"
-url = "https://environments.fission.io/"
 weight = 12
 [[menu.main]]
+name = "Examples"
```

Comment:
I dont see examples link on homepage somehow. Dont know if its caching issue or if we manually need to add link on homepage ?

Reply:
I had the cache issue as well. Clearing cache should fix the issue.